docs/pages/project/contributing/contributing-error.md

```diff
@@ -145,7 +145,7 @@ func main() {
 		DebugLevel: true,
```

Comment:
well issue also points to hint in example for `LogLevel`.

Reply:
https://github.com/meshery/meshery/blob/6889021a59c16de63386bdb72f37d2c0b77b14c8/server/cmd/main.go#L77